### DISCUSSION TITLE
Elaborate error in the `error` object of `useFetch` return

### DISCUSSION THREAD

**Discussion by iannngit** [nuxt/nuxt#17500] [Ideas]
```
const { data, pending, error, refresh } = await useFetch(API_ENDPOINT);
```

the `error` object is an Object or boolean according to the doc. But what I've found is it's always true or false, which makes it hard for debugging. 

Also, if errors occur in useFetch, can the errors surface to client side or pop to terminal?

My workaround is when error is true then I call the API directly to check the response.

**Comment by wobsoriano** [comment]
https://github.com/nuxt/framework/discussions/5236#discussioncomment-2873301
